packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -86,7 +86,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
             { renderedItems.map(
               (item, index) => (
                 <li className='ms-Breadcrumb-listItem' key={ item.key || String(index) } ref={ item.key || String(index) }>
-                  { this._renderItem(item) }
+                  { this.props.onRenderItem ? this.props.onRenderItem(item) : this._renderItem(item) }
```

Comment:
this will have to change a bit with `IRenderFunction`, basically we want to pass in the default rendering, in case the user wants to just augment the existing rendering